web/src/components/navbar/NavBarDesktop.tsx

```diff
@@ -73,7 +75,16 @@ export const NavBarDesktop = (): ReactElement => {
     <div className="position-sticky top-0 z-500 bg-white">
       <nav aria-label="Primary" className="grid-container-widescreen desktop:padding-x-7">
         <div className="display-flex flex-row flex-justify flex-align-center height-8">
-          <NavigatorLogo />
+          <div className="display-flex flex-row flex-align-center">
```

Comment:
[sand] Kind of nit-picky - the focus box doesn't actually surround the image on click. It'd be nice to fix this.
<img width="308" alt="Screenshot 2024-01-29 at 4 59 29 PM" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22647707/6cb52147-8cf2-40ca-9fed-494ef924a877">

Reply:
Not specifically part of the AC, but it might be nice to have this use the blue color if it's not a PITA.